ci/launch_cluster.sh

```diff
@@ -38,25 +38,34 @@ envsubst <<EOF > "$CONFIG_PATH"
 ---
 launch_config_version: 1
 deployment_name: $DEPLOYMENT_NAME
-installer_url: $INSTALLER
-provider: onprem
-platform: aws
 aws_region: us-west-2
-os_name: cent-os-7-dcos-prereqs
-key_helper: true
+installer_url: $INSTALLER_URL
```

Comment:
Please move up to limit the number of line changes.

Reply:
Done